lib/Common/Memory/LargeHeapBlock.h

```diff
@@ -29,11 +29,11 @@ struct LargeObjectHeader
 #endif
 
 public:
-    bool markOnOOMRescan;
+    bool markOnOOMRescan:1;
```

Comment:
This structure appears to have alignment concerns before- can you confirm that the alignment hasn't changed? (see use of unused2/unused3/unused4)